apps/aecore/lib/aecore/chain/block_validation.ex

```diff
@@ -8,7 +8,7 @@ defmodule Aecore.Chain.BlockValidation do
   alias Aecore.Chain.ChainState
   alias Aecore.Chain.Difficulty
 
-  @spec calculate_and_validate_block!(Block.t(), Block.t(), map(), list(Block.t())) :: {:error, term()} | :ok
+  @spec calculate_and_validate_block!(Block.block(), Block.block(), map(), list(Block.block())) :: {:error, term()} | :ok
```

Comment:
why? we prefer `.t()` notation

Reply:
The `.t()` notation (the missing `-type` in `Aecore.Structures.Block`) is introduced by an unmerged PR (#182). We can merge that one first and then revert this commit.